vendor/engines/secure_rooms/app/models/secure_rooms/occupancy.rb

```diff
@@ -5,6 +5,8 @@ class Occupancy < ActiveRecord::Base
     belongs_to :secure_room, foreign_key: :product_id
     belongs_to :user
     belongs_to :account
+    belongs_to :order
```

Comment:
I don't think we need a direct relation to the `order`. If we need to get at the order, we can get at it from the `order_detail.order`.